www/src/utils/sidebar/item-list.js

```diff
@@ -43,9 +43,23 @@ const itemListTutorial = extendSidebarData(tutorialSidebar)
 const itemListContributing = extendSidebarData(contributingSidebar)
 const itemListFeatures = extendSidebarData(featuresSidebar)
 
+const itemListLookup = {
+  docs: itemListDocs,
+  contributing: itemListContributing,
+  tutorial: itemListTutorial,
+  features: itemListFeatures,
+}
+
+function getItemList(path) {
+  // FIXME work with localized paths
+  const [urlSegment] = path.split("/").slice(1)
```

Comment:
Pending [this change](https://github.com/gatsbyjs/gatsby/pull/20828/files#diff-83a577600c100cb2121f335171086e74R51)